db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -712,6 +726,23 @@ class NonBatchedOpsStressTest : public StressTest {
                   Slice(value).ToString(true /* hex */).c_str());
           is_consistent = false;
         }
+
+        if (use_txn) {
+          assert(!read_your_write_values.empty());
+          if (read_your_write_values[i] != no_ryw_check &&
+              read_your_write_values[i] != values[i].ToString()) {
+            fprintf(stderr,
+                    "MultiGet / Get returned different results with key %s\n",
+                    keys[i].ToString(true).c_str());
+            fprintf(stderr, "MultiGet / Get returned value %s\n",
+                    values[i].ToString(true).c_str());
```

Comment:
Suggested message strings:

- "MultiGet returned value different from what was written for key %s\n"
- "MultiGet returned value %s\n"

Reply:
Thank you for the suggestion. I have updated the error messages.